docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
```

Comment:
Wanna add a docker compose file for build as well?

Reply:
What's the use case? I've scoped the compose to dev mode only.